.github/workflows/update_infra.yml

```diff
@@ -0,0 +1,63 @@
+name: Open a PR in Infra repository
+
+on:
+  pull_request:
+    branches:
+      - main
+    types: [ closed ]
+    paths:
+      - 'gpd/openapi/openapi.json'
+      - 'payments/openapi/openapi.json'
+      - 'reporting-analysis/openapi/openapi.json'
+
+
+  workflow_dispatch:
+
+jobs:
+  pull-request:
+    if: ${{ github.event.pull_request.merged }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      # set repo
+      - name: Set GPD repo
+        run: |
+          echo "REPO=gpd" >> $GITHUB_ENV
```

Comment:
correct ?
<img width="399" alt="image" src="https://user-images.githubusercontent.com/36746022/170668390-cf905fd9-62dd-4ed2-b4c4-736227922669.png">

Reply:
yes, for the moment